src/utils/__init__.py

```diff
@@ -1,5 +1,5 @@
-from utils.registry import MetaParent
-from utils.tensorboards import GLOBAL_TENSORBOARD_WRITER, TensorboardWriter
+from .registry import MetaParent
```

Comment:
Let's write relative paths only in \_\_init\_\_.py files. 

Quote from PEP8 guidelines: "Absolute imports are recommended, as they are usually more readable and tend to be better behaved (or at least give better error messages) if the import system is incorrectly configured (such as when a directory inside a package ends up on sys.path)". 

```suggestion
from utils.registry import MetaParent
```